Dockerfile

```diff
@@ -68,19 +57,13 @@ RUN chmod +x /home/akeneo/pim-docs/build.sh && \
     wget https://github.com/akeneo/pim-community-dev/archive/master.zip -P /home/akeneo/pim-docs/ && \
     unzip /home/akeneo/pim-docs/master.zip -d /home/akeneo/pim-docs/ && \
     #
-    # Install Akeneo PIM
+    # Install vendors
     cd /home/akeneo/pim-docs/pim-community-dev-master/ && \
     php -d memory_limit=3G /home/akeneo/pim-docs/composer.phar install --no-dev --no-suggest --ignore-platform-reqs
 
-COPY docker/wait_for_mysql.sh /wait_for_mysql.sh
-RUN service mysql start && chmod +x /wait_for_mysql.sh && /wait_for_mysql.sh && \
-    mysql -u root -e "CREATE DATABASE akeneo_pim" && \
-    mysql -u root -e "GRANT ALL PRIVILEGES ON akeneo_pim.* TO akeneo_pim@localhost IDENTIFIED BY 'akeneo_pim'" && \
-    cp /home/akeneo/pim-docs/pim-community-dev-master/app/config/parameters.yml.dist /home/akeneo/pim-docs/pim-community-dev-master/app/config/parameters.yml && \
-    cd /home/akeneo/pim-docs/pim-community-dev-master/ && php bin/console doctrine:schema:create --env=prod && \
-    cd /home/akeneo/pim-docs/pim-community-dev-master/ && php bin/console pim:installer:assets --env=prod && \
+RUN cd /home/akeneo/pim-docs/pim-community-dev-master/ && php bin/console pim:installer:assets --env=prod && \
     cd /home/akeneo/pim-docs/pim-community-dev-master/ && sed -i "s#replace: '/bundles'#replace: '../bundles'#" frontend/build/compile-less.js && \
-    cd /home/akeneo/pim-docs/pim-community-dev-master/ && yarn install && yarn less
+    cd /home/akeneo/pim-docs/pim-community-dev-master/ && yarnpkg install && yarnpkg less
```

Comment:
Do you still need yarn ?

Reply:
I still launch the `yarn less` command
not sure it's needed
I'll test tomorrow